promgen/static/js/promgen.vue.js

```diff
@@ -22,12 +22,8 @@ const dataStore = Vue.reactive({
 
 const silenceStore = Vue.reactive({
     state: {
-        show: false,
         labels: {}
     },
-    showForm() {
-        this.state.show = true;
-    },
```

Comment:
GitHub does not allow me to make code suggestions if there are deleted lines in your changes, so I will write the desired result here:

```
 const silenceStore = Vue.reactive({
     state: {
         show: false,
         labels: {}
     },
     showModal() {
         this.state.show = true;
     },
     hideModal() {
         this.state.show = false;
     },
```

That will allow us to control the visibility of the modal via the store, which can be accessed from any component.